triage-process.md

```diff
@@ -0,0 +1,59 @@
+# High Level Proposal:
+1. Define and publish a process to handle issues in a structured manner.
+    1. Issue Handling Process
+    1. Issue Stages
+    1. Roles
+    1. SLA
+
+## Process Definition
+
+### Issue Handling Process
+1. All incoming issues and PR’s should be as. These are items that a Triager would look at on a daily basis
+1. Every issue triaged should be assessed for its workability. If assessing the workability of the issue requires more information, the Triager would collaborate with the OP until workability is determined.
+    1. Workability Classification:
+        1. New Feature Request
+        1. Enhancement to Existing Feature
+        1. Bug
+1. Workable Issues should be prioritized and Labeled as one of 3 priorities
+    * P0
+    * P1
+    * P2
+1. Prioritized Issues (and unworked) issues should be labeled as ‘Ready for Work’. This should signal to any contributor willing to help that they can work on this issue.
+1. Once a contributor is working on this issue, the issue should be:
+    1. Assigned to him
+    1. Labeled as ‘In Progress’
+1. Once work is complete and issue has been resolved it should be labeled as ‘Completed’ and Issue should be closed. 
```

Comment:
'Completed' label is not required. If the issue is closed and there is a commit associated with it then it is completed.

Reply:
Same as above - Its not a label. I think you might be reading the column incorrectly. See action lablel column far right in the same row.

Reply:
same as above. Perhaps, rephrase it to
```
Once work is complete and the issue has been resolved, simply close the Issue. A closed issue with associated merged PR is considered as "Complete".
```

Reply:
done.